cli/src/commands/bookmark/move.rs

```diff
@@ -56,14 +56,14 @@ pub struct BookmarkMoveArgs {
 
     /// Move bookmarks to this revision
     // We intentionally do not support the short `-t` for `--to` since we don't
-    // support `-f` for `--from`.
+    // support `-f` for `--from`. Currently this defaults to the working copy, but in the near
+    // future it will be required to explicitly specify it.
```

Comment:
nit: Reflow text. I think if you simply join the lines, then `rustfmt` will take for wrapping them correctly.

Reply:
Done, I think. Neither `jj fix` nor `cargo +nightly fmt` wrapped the text, so I wrapped it manually. I did not try invoking `rustfmt` directly though. Should I do that instead?